<launch>

    <node name="pcl_colored_node" pkg="myproject1" type="pcl_colored_node" output="screen">

        <remap from="input_cloud" to="/sensor/laser/vlp16/front/pointcloud_xyzi"/>
        <remap from="camera_info" to="/sensor/camera/blackfly/camera_info"/>
        <remap from="input_image" to="/sensor/camera/blackfly/image_rect_color"/>

        <remap from="output_cloud" to="/sensor/laser/vlp16/front/point_cloud_colored"/>
    </node>
</launch>